scripts/train_lm.py

```diff
@@ -63,6 +68,8 @@
                     help="Directory containing the training text.")
 parser.add_argument("order",
                     help="Order of N-gram model to be trained.")
+parser.add_argument("work_dir",
+                    help="Working directory for building the language model.")
 parser.add_argument("lm_dir",
```

Comment:
Ok. And do i need to revert the changes in run.sh to used the default value in poco egs?